server/pbench/bin/pbench-unpack-tarballs

```diff
@@ -192,12 +216,9 @@ while read size result ;do
     prefix=$(getconf.py -C $UNPACK_PATH/$hostname/$resultname/metadata.log prefix run)
     user=$(getconf.py -C $UNPACK_PATH/$hostname/$resultname/metadata.log user run)
 
-    # Version 001 agents use a prefix file
-    # If there is a prefix file, create a link as specified by the
-    # prefix.  Even if no prefix is specified, we still want to link
-    # from the results/ hierarchy to incoming/.
-    basedir=$(dirname $link)
-    # pbench-dispatch has already moved it to the .prefix subdir
+    # Version 001 agents use a prefix file.  If there is a prefix file,
+    # create a link as specified in the prefix file.  pbench-dispatch
+    # has already moved it to the .prefix subdir
```

Comment:
Is `basedir` defined somewhere else? If not, we still need to define it: it is used on line 222.

Reply:
Yes, I think it is defined at [line 139](https://github.com/distributed-system-analysis/pbench/pull/935/files/016d4b7a155be09dc052c65c4c935fa676896528#diff-1e38c99b24d595c1a1a6f1685f2f6b55R139).